.buildkite/pipeline.yml

```diff
@@ -239,6 +239,20 @@ steps:
         agents:
           slurm_mem: 20GB
 
+      - label: ":computer: no lim ARS baroclinic wave (ρe) equilmoist with EDMFX"
```

Comment:
Could we have a different symbol for EDMFX jobs so that they are easier to find in the CI list?

Reply:
How does :genie: look?